openerp-atomfeed-service/src/main/java/org/bahmni/feed/openerp/domain/OpenMRSResource.java

```diff
@@ -0,0 +1,52 @@
+package org.bahmni.feed.openerp.domain;
+
+import java.util.Date;
+
+public class OpenMRSResource {
+
+    private String id;
+    private Date dateCreated;
+    private Date lastUpdated;
+    private String name;
+    private Boolean isActive;
```

Comment:
stick to java name convention  to "active"? you will need to add  
annotation over the setter method like (courtesy the contract) 
   
    @JsonProperty("isActive")
    public void setActive(boolean active) {
        this.active = active;
    }